Code_Exercises/Exercise_02_Hello_World/solution.cpp

```diff
@@ -11,7 +11,11 @@
 #define CATCH_CONFIG_MAIN
 #include <catch2/catch.hpp>
 
+#if __has_include(<sycl/sycl.hpp>)
 #include <sycl/sycl.hpp>
+#else
+#include <CL/sycl.hpp>
+#endif
```

Comment:
sorry you had to move this back.
I am not sure which version of DPCPP is still using the old style. But keeping this is safer anyway so we become independent from the implementation.

Reply:
No prob